src/client/renderer/shaders/default_fs.glsl

```diff
@@ -123,8 +124,6 @@ void LightFragment(in vec4 diffuse, in vec3 lightmap, in vec3 normalmap, in floa
 			}
 		}
 	}
-
-	light *= MODULATE_SCALE;
```

Comment:
I think it's actually cleaner to keep this here, rather than munged in the line below.

Reply:
It's here because tonemap breaks it. So it doesn't break an equation if I move it to the end of file.

Reply:
👍